easybuild/easyconfigs/d/dotNET-SDK/dotNET-SDK-3.1.300-GCCcore-10.1.0.eb

```diff
@@ -0,0 +1,33 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 	https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::	Robert Qiao <rob.qiao@flinders.edu.au>
+# License::	MIT
+#
+# Notes::	requires glibc v2.14 or higher
+##
+
+easyblock = 'Tarball'
+
+name = 'dotNET-SDK'
+version = '3.1.300'
```

Comment:
I think a `versionsuffix = '-rhel.6-x64'` is appropriate for this binary (to indicate which is used)